src/main/resources/deployments/defaultDeployment.properties

```diff
@@ -198,6 +198,15 @@ dmptool.base.url=https://dmptool-stg.cdlib.org
 # For testing locally, e.g. https://92832de0.ngrok.io, defaults to properties.getServerUrl()
 dmptool.callback.base.url=
 
+# Client ID / Secret from Bitwarden
+dcd.client.id=kW1WtsEeW7b9LugvT-jq9148gmkTPDeFFZe4NM9mV1k
+dcd.client.secret=Q5QHqNn91Q2tvJwsqkRDorAS9i6Mg685GLY6TYLBWH22MITQ4CvOnRx3kQ43IKAr
```

Comment:
remove the secrets and move them to the closed sources

Reply:
ok, although in principle it's too late now - the credentials are already public from the moment you pushed to non-private repository/branch. It's fine here because I don't think there is much value in this dcd client.id/secret, but try not doing that next time!

Reply:
removed